src/main/scala/com/redislabs/provider/redis/redisFunctions.scala

```diff
@@ -272,6 +272,18 @@ class RedisContext(@transient val sc: SparkContext) extends Serializable {
     kvs.foreachPartition(partition => setHash(hashName, partition, ttl, redisConfig, readWriteConfig))
   }
 
+  /**
+    * @param kvs      Pair RDD of hashName/K/V
+    * @param ttl      time to live
```

Comment:
Can you add what the time resolution for TTL is? e.g. seconds, ms, etc

Reply:
@jessicastewart-udemy  It is the second, I will update.